portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,181 @@
 <!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+<html lang="es">
+    <head>
+        <meta charset="UTF-8">
+        <title>Andrew Hojel</title>
+        <link href="https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@300;400;500&display=swap" rel="stylesheet">
+        <link rel="stylesheet" href="//use.fontawesome.com/releases/v5.0.7/css/all.css">
+        <link rel="stylesheet" href="style.css">
+        <link rel='icon' href='favicon.png' type='image/x-icon'/ >
+        <script src="script.js"></script>
+    </head>
+    <body> <!-- onload="writeSnippets()"> -->
+        <div id="content">
+
+            <!-- Profile & Introduction Section -->
+            <img id="profile" src="images/profile_picture.jpg" alt=""/>
```

Comment:
Same comment on the trailing slash, here and elsewhere below.  (Tags like `img` are "self-closing" in HTML, which means they are automatically considered to be closed once you type the final `>` of the tag.  XML doesn't have these kinds of tags, so they introduced the `/>` shorthand to make it less tedious to write.)